RevenueCatUI/CustomerCenter/Views/SubscriptionDetailView.swift

```diff
@@ -40,7 +41,7 @@ struct SubscriptionDetailView: View {
     @Environment(\.supportInformation)
     private var support
 
-    @StateObject
+    @ObservedObject
```

Comment:
What's the reason for this one?

Reply:
Same reason as the one that we have [here](https://github.com/RevenueCat/purchases-ios/pull/5108#discussion_r2167663358)!